config/routes.rb

```diff
@@ -4,4 +4,7 @@
       get :payload
     end
   end
+
+  post '/affirm/confirm', to: "affirm#confirm", as: :confirm_affirm
```

Comment:
These routes don't point anywhere as of this commit. They should be included in the commit that impliments their functionality, ideally one by one.